Policies/ContainerRegistry/Azure_ContainerRegistry_Config_Enable_Security_Scanning/Security - Container Registry - DeploySecurityScannerIdentity.json

```diff
@@ -1,19 +1,19 @@
 {
   "mode": "All",
   "parameters": {
-	"effect": {
-	  "type": "String",
-	  "metadata": {
-		"displayName": "Effect",
-		"description": "Enable or disable the execution of the policy"
-	  },
-	  "allowedValues": [
-		"DeployIfNotExists",
-		"AuditIfNotExists",
-		"Disabled"
-	  ],
-	  "defaultValue": "DeployIfNotExists"
-	},
+		"effect": {
```

Comment:
Previously we have allowed MG level RBAC assignment as well, that check is missing here.